packages/squiggle-lang/__tests__/Distributions/Dotwise_test.res

```diff
@@ -0,0 +1,50 @@
+open Jest
+open Expect
+open TestHelpers
+open FastCheck
+open Arbitrary
+open Property.Sync
+
+describe("dotSubtract", () => {
+  test("mean of normal minus exponential (unit)", () => {
+    let mean = 0.0
+    let rate = 10.0
+
+    let dotDifference = DistributionOperation.Constructors.pointwiseSubtract(
+      ~env,
+      mkNormal(mean, 1.0),
+      mkExponential(rate),
+    )
+    let meanResult = E.R2.bind(DistributionOperation.Constructors.mean(~env), dotDifference)
+    let meanAnalytical = mean -. 1.0 /. rate
```

Comment:
where does this math come from? Really not intuitive to me.

Reply:
observed in playground. plausibly a coincidence, because property test didn't really perform super well.

Reply:
it's just `mean(a) - mean(b)`, for normal a and exponential b

Reply:
Can you call that directly? Like, ``Symbolic.exponential.mean(a)``? I don't like re-calculating it here.

Reply:
done.